lib/svg.js

```diff
@@ -5,7 +5,10 @@ const Node = require('./node');
 const {allCategories, allTypes} = require('./utils/nodeTypes');
 
 // Parser
-const parser = new xml2js.Parser();
+const parser = new xml2js.Parser({ 
+  preserveChildrenOrder: true, 
```

Comment:
Not sure if this will be a breaking change for anyone currently using this library with preserveChildrenOrder defaulted to `false`

Reply:
It should only affect the cases where people rely on a wrong order. Since it's a wrong behavior and the SVGI API is the same, I would move forward with it :)